client/src/utils/card.ts

```diff
@@ -0,0 +1,6 @@
+export const currencyFormatter = new Intl.NumberFormat(undefined, {
```

Comment:
Will this allow different currencies to be used? Also helpful for conversion maybe?

Reply:
I haven't done any research yet on currency conversion in js, but I'll definitely keep note of this and make some improvements in the future

Reply:
Sounds great!